src/basicview.cpp

```diff
@@ -152,17 +155,21 @@ void BasicView::update(Cluster cluster)
     bool isRunning = cluster.status() == "Running";
     bool isPaused = cluster.status() == "Paused";
     topStatus->setText(cluster.status());
+    serviceButton->setEnabled(isRunning || isPaused);
     stopButton->setEnabled(isRunning || isPaused);
     pauseButton->setEnabled(isRunning || isPaused);
-    mountButton->setEnabled(isRunning || isPaused);
     deleteButton->setEnabled(exists);
     dashboardButton->setEnabled(isRunning);
 #if __linux__ || __APPLE__
-    dockerEnvButton->setEnabled(isRunning);
-    sshButton->setEnabled(exists);
+    dockerEnvButton->setEnabled(isRunning || isPaused);
+    sshButton->setEnabled(isRunning || isPaused);
+    mountButton->setEnabled(isRunning || isPaused);
+    tunnelButton->setEnabled(isRunning || isPaused);
```

Comment:
Doesn't have to be done in this PR, but we should make a variable called `isRunningOrPaused` instead of repeatedly doing `isRunning || isPaused`

Reply:
good idea